identity/client/.env

```diff
@@ -0,0 +1 @@
+VITE_APP_VERSION=$npm_package_version
```

Comment:
I haven't looked at the infra around this but is this something that we have to manually manage? Is this set somewhere on release?

Reply:
It comes from the package.json file (where it's hardcoded). I've taken the same approach as [operate](https://github.com/camunda/camunda/blob/fc38c934dbb43f2b67bb10b2fdfada3dcdee2315/operate/client/.env#L1) and [tasklist](https://github.com/camunda/camunda/blob/fc38c934dbb43f2b67bb10b2fdfada3dcdee2315/tasklist/client/.env#L1) here.